app/src/main/AndroidManifest.xml

```diff
@@ -38,6 +38,16 @@
                 android:value="" />
         </activity>
 
+        <activity
+            android:name=".ui.screens.newsbrainz.NewsBrainzActivity"
+            android:exported="false"
+            android:label="@string/title_activity_news_brainz"
+            android:theme="@style/AppTheme">
+            <meta-data
```

Comment:
This is not a shared library or anything so we don't need this tag here. Please remove it.